common4j/src/main/com/microsoft/identity/common/java/crypto/StorageEncryptionManager.java

```diff
@@ -199,8 +199,14 @@ public byte[] decrypt(final byte[] cipherText) throws ClientException {
         }
 
         final List<AbstractSecretKeyLoader> keysForDecryption = getKeyLoaderForDecryption(cipherText);
-        if (keysForDecryption == null || keysForDecryption.size() == 0) {
-            throw new IllegalStateException("KeyLoader list must not be null or empty.");
+        if (keysForDecryption == null) {
```

Comment:
I'm thinking maybe expose the settings as "shouldNotCrashIfDataNotEncrypted", and then use that flag here.

The other method we can keep it align with the lombok nonnull identifier.